sphinx/writers/latex.py

```diff
@@ -901,7 +894,6 @@ def visit_document(self, node):
         self.curfilestack.append(node.get('docname', ''))
         if self.first_document == 1:
             # the first document is all the regular content ...
-            self.body.append(BEGIN_DOC % self.elements)
             self.first_document = 0
         elif self.first_document == 0:
```

Comment:
Finally I found how to enter this if-block:  `latex_appendicies`. `latex_documents` is not related with this.
So additional testcases are not needed for this change (including `test_build_multiple_latex_docs`).

Sorry for my wrong comment above. I also have not knew about this feature at all...

Reply:
So, to be sure I get it correctly regarding the tests : I will remove tests regarding ``test_build_multiple_latex_docs`` but do I remove the ones I talk about in the upper comment ?

Reply:
Yes, please :-)

Reply:
however, there is no test about appendix. It would better to add somes testcases and then refactor the if-block in order to check the changes.

Reply:
+1